DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/traffic/ixnetwork/ixnetwork_ixia_client_impl.py

```diff
@@ -603,3 +606,23 @@ def run_send_arp(self, device, command, *argv, **kwarg):
         return err, [{"success": msg["arg2"],
                       "port": msg["port"],
                       "src_ip": msg["src_ip"]} for msg in res]
+
+    def run_update_l1_config(self, device, command, *argv, **kwarg):
+        speeds_map = {"1000": "speed1000",
+                      "100": "speed100fd",
+                      "10000": "speed10g",
+                      "2500": "speed2.5g",
+                      "5000": "speed5g"}
+
+        if not IxnetworkIxiaClientImpl.ixnet:
+            return 0, "Ixia not connected"
+        if command == "update_l1_config":
+            vports = IxnetworkIxiaClientImpl.ixnet.Vport.find()
+            speed = kwarg['params'][0]["speed"]
+            autoneg = kwarg['params'][0].get("autoneg", True)
+            for vport in vports:
```

Comment:
Can we change the speed mode only on a single port?

Reply:
not at the moment..